src/test/java/net/smartcosmos/dto/metadata/MetadataCreateTest.java

```diff
@@ -0,0 +1,54 @@
+package net.smartcosmos.dto.metadata;
```

Comment:
Coverage for the objects show that one method is missing but since the code is generated from the Lombok annotations which method is not clear.  I suspect more than one is missed, perhaps `equals()` or the constructor?